core/chains/evm/client/errors.go

```diff
@@ -483,6 +490,10 @@ func ClassifySendError(err error, lggr logger.SugaredLogger, tx *types.Transacti
 		), "err", sendError, "etx", tx)
 		return commonclient.InsufficientFunds
 	}
+	if sendError.IsServiceUnavailable() {
+		lggr.Errorw("service unavailable while sending transaction %x", tx.Hash(), "err", sendError, "etx", tx)
+		return commonclient.Retryable
+	}
```

Comment:
Could you also please change the last log line in this file to a critical error?
Here: "lggr.Errorw("Unknown error encountered when sending transaction", "err", err, "etx", tx)"

That error should be critical!